### PR TITLE
Added loading spinner animation

### LINKED ISSUE #364: Add loading spinner
Search is nice and snappy. On slow connections, a spinner to indicate something is happening would be useful. 

~~depends on https://github.com/mozilla/network-pulse/issues/361~~

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #364

### CODE REVIEW

File: components/project-list/project-list.jsx [hunk 1]
```diff
@@ -80,24 +80,33 @@ export default React.createClass({
     this.setState({displayBatchIndex: this.state.displayBatchIndex+1});
   },
   render() {
-    let projects = this.state.dataLoaded ? this.applyFilterToList(this.state.entries,this.props.params) : null;
+    let projects = (<div className="loading my-5 d-flex justify-content-center align-items-center">
+                      <div></div>
```
Comment (Pomax) [inline components/project-list/project-list.jsx]: why are there three empty divs here?

Comment (mmmavis) [inline components/project-list/project-list.jsx]: one `div` for each dot

Comment (mmmavis) [inline components/project-list/project-list.jsx]: loading-animation

Comment (Pomax) [inline components/project-list/project-list.jsx]: Oh, I see! that is some sneaksy CSS =)

**Comment by Pomax** [comment]
R+

File: components/project-list/project-list.jsx (hunk 1 above)
Comment (mmmavis) [inline components/project-list/project-list.jsx]: 💃

**Pull request merged**